jerry-core/debugger/debugger.h

```diff
@@ -130,16 +132,17 @@ typedef enum
   JERRY_DEBUGGER_STOP = 5, /**< stop execution */
   JERRY_DEBUGGER_CLIENT_SOURCE = 6, /**< first message of client source */
   JERRY_DEBUGGER_CLIENT_SOURCE_PART = 7, /**< next message of client source */
+  JERRY_DEBUGGER_NO_MORE_SOURCE = 8, /**< end message of client source */
```

Comment:
JERRY_DEBUGGER_NO_MORE_SOURCES and `/* no more sources */` I think `message` is unnecessary since other comments does not include it (these are obviously messages).